projects/kit/components/input-file/input-file.template.html

```diff
@@ -39,11 +39,11 @@
             [accept]="accept"
             [multiple]="multiple"
             [tuiFocusable]="focusable"
-            (change)="onFilesSelected(input, texts, units)"
+            (change)="onFilesSelected(input, $any(texts), units)"
```

Comment:
Again, what's the issue with translations type?

Reply:
<img width="1274" alt="image" src="https://user-images.githubusercontent.com/12021443/156890072-7425afe9-6ae4-4085-b7e1-823146ff2d13.png">

Reply:
fixed